tensorboard/plugins/projector/projector_plugin.py

```diff
@@ -471,6 +478,14 @@ def _append_plugin_asset_directories(self, run_path_pairs):
       assets_path_pair = (run, os.path.abspath(assets_dir))
       run_path_pairs.append(assets_path_pair)
 
+  @wrappers.Request.application
+  def _serve_file(self, file_path, request):
```

Comment:
I thought about that but could not think of a good place for this utility. I am currently leaning towards putting this somewhere in the base_plugin but I don't know how I feel about that. The major issue with putting this under http_util is the fact that it will need to take the asset_provider as an input and also become a bit stateful for memoization or up front gzipping. In my naive view, utils should try to remain very stateless. 

Anyways, since there is no good candidate for dynamic-plugification immediately, we have some time to do so and was going to take an attempt when we need that.

Btw, I did try just doing 302, from this endpoint, to /file/path but that changed the base URL of the iframe which was undesirable.